real_estate/models/estate_property.py

```diff
@@ -0,0 +1,108 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Property"
+    _order = 'id desc'
+
+    # Description
+    name = fields.Char(string="Property Name", required=True)
+    description = fields.Text(string="Property Description")
+    postcode = fields.Char(string="Postcode") 
+    date_availability = fields.Date(string="Availabile From",copy=False,
+        default=lambda self:fields.Datetime.today() + relativedelta(months=3)
+    )
+
+    expected_price = fields.Float(string="Expected Price",required=True)
+    selling_price = fields.Float(string="Selling Price",readonly=True, copy=False)
+    bedrooms = fields.Integer(string="Bedrooms",default=2)
+    living_area = fields.Integer(string="Living Area")
+    facades = fields.Integer(string="Facades")
+    garage = fields.Boolean(string="Garage")
+    garden = fields.Boolean(string="Garden")
+    garden_area = fields.Integer(string="Garden Area")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ('north', "North"),
+            ('south', "South"),
+            ('east', "East"), 
+            ('west', "West")
+        ]
+    )
+    total_area = fields.Float(compute='_compute_total_area')
+    active = fields.Boolean(string="Active", default=True)
+    state = fields.Selection(string='State',
+        selection=[
+            ('new', "New"),
+            ('offer_received', "Offer Received"),
+            ('offer_accepted', "Offer Accepted"),
+            ('sold', "Sold"),
+            ('cancelled', "Cancelled")
+        ],
+        required=True,default="new",copy=False
```

Comment:
```suggestion
        required=True, default="new", copy=False
```
exactly one space after `,`